Vagrantfile

```diff
@@ -30,15 +29,27 @@ Vagrant::configure("2") do |config|
 
   config.vm.define "empty", autostart: false
 
+  # For normal users
```

Comment:
Note: If anything this should either:

- be reversed
- just autodetect if you are on windows and don't sync the directory in that case.